framework_tvb/tvb/interfaces/rest/client/examples/launch_operation.py

```diff
@@ -125,26 +120,33 @@
             break
         time.sleep(5)
     logger.info("The region mapping uploading has finished with status: {}".format(status))
-    os.remove(connectivity_path)
-    os.remove(surface_path)
 
-    logger.info("Preparing a connectivity csv H5 file (this one requires two datatype files: one containing weights and the other one containing tracts)...")
+    logger.info("Downloading the region mapping uploaded above")
+    region_mapping_gid = tvb_client.get_operation_results(operation_gid)[0].gid
+    region_mapping_file_path = tvb_client.retrieve_datatype(region_mapping_gid, tvb_client.temp_folder)
+    with RegionMappingH5(region_mapping_file_path) as rm:
+        region_mapping_connectivity_gid = rm.connectivity.load().hex
+    logger.info("Region mapping with gid {} is linked to a connectivity with gid {}".format(region_mapping_gid,
+                                                                                            region_mapping_connectivity_gid))
+
+    logger.info(
+        "Preparing a connectivity csv H5 file (this one requires two datatype files: one containing weights and the other one containing tracts)...")
     csv_view_model = CSVConnectivityImporterModel()
 
-    csv_weights_path = os.path.join(os.path.dirname(tvb_data.__file__), 'dti_pipeline_toronto', 'output_ConnectionCapacityMatrix.csv')
-    csv_tracts_path = os.path.join(os.path.dirname(tvb_data.__file__), 'dti_pipeline_toronto', 'output_ConnectionDistanceMatrix.csv')
+    csv_weights_path = os.path.join(os.path.dirname(tvb_data.__file__), 'dti_pipeline_toronto',
+                                    'output_ConnectionCapacityMatrix.csv')
+    csv_tracts_path = os.path.join(os.path.dirname(tvb_data.__file__), 'dti_pipeline_toronto',
+                                   'output_ConnectionDistanceMatrix.csv')
```

Comment:
This example has become too complex and very difficult to read.
What is the purpose of enumerating all these importers?
If we need all for some reason, at least we should create separate functions for each of the calls